packages/source-iotsitewise/src/time-series-data/client/client.spec.ts

```diff
@@ -38,6 +40,7 @@ describe('getHistoricalPropertyDataPoints', () => {
         end: endDate,
         resolution: '0',
         fetchFromStartToEnd: true,
+        aggregationType: AGGREGATE_TYPE,
```

Comment:
is there a way to write this so we don't need the `aggregationType` since resolution is 0 here?